pynance/data/retrieve.py

```diff
@@ -10,14 +10,14 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, start, end = datetime.today().date().isoformat()):
```

Comment:
This should be end=None. Then test inside the function for None and replace it. 

The default value should be at least 1 day after today in order to account for potential time zone differences. It doesn't even matter if it's a long time afterward (e.g., '2017' right now). 

For performance purposes, I'd look at the type of time object that is actually being used under the hood (pd.Timestamp, I think) and use that data type. Then there's no unnecessary conversion from one type to another.